assets/js/screens/Collection/List.test.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
For this component test, seems like the component itself does the following:

- Renders out "hero" (this will change to "header" or something similar with the new designs, but easy enough to update later) content.
- Renders out a "Create" button.
- Renders out a collection list section
- Renders out individual list items (`CollectionListRow`)

Can you add some extra tests to cover the general functionality above, if any are missing?

Reply:
I already renders collection list and hero sections, I added create button and list item tests.